models/OrderModel.ts

```diff
@@ -29,6 +30,11 @@ export class OrderModel extends BaseEntity {
   @Index('recent_orders_index')
   orderedAt: Date;
 
+  @ManyToOne((type) => OrderPickupEventModel, (pickupEvent) => pickupEvent.orders, { eager: true, nullable: false })
```

Comment:
Are these non-null by default like other properties?

Reply:
It seems to be `nullable: true` by default, but for some reason I can't find any documentation on it. All our other `@ManyToOne` usages include `nullable: false`, and so do the ones online, but again nowhere on TypeOrm docs do I see it